libs/grok/src/main/java/org/elasticsearch/grok/Grok.java

```diff
@@ -101,7 +106,8 @@ private Grok(Map<String, String> patternBank, String grokPattern, boolean namedC
 
         String expression = toRegex(grokPattern);
         byte[] expressionBytes = expression.getBytes(StandardCharsets.UTF_8);
-        this.compiledExpression = new Regex(expressionBytes, 0, expressionBytes.length, Option.DEFAULT, UTF8Encoding.INSTANCE);
+        this.compiledExpression = new Regex(expressionBytes, 0, expressionBytes.length, Option.DEFAULT, UTF8Encoding.INSTANCE,
+            message -> logger.log(Level.DEBUG, message));
```

Comment:
^^ this line is only substantive change here. (it implements the callback on the message)

Reply:
Thanks Jake! I'm happy that this is an easy change.